### DISCUSSION TITLE
Welcome to django-breeze Discussions!

### DISCUSSION THREAD

**Discussion by Louxsdon** [Louxsdon/django-breeze#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by diegoquiroz** [comment]
Hi @Louxsdon, thanks for building Django Breeze. I just made 2 PRs to support Django 5 and Svelte.
- #8 
- #7

**Reply by Louxsdon** [reply]
Thanks, @diegoquiroz for the contributions.

I will go through then merge for later release.

**Comment by mrfolksy** [comment]
Hi @Louxsdon, great project, I used it to bootstrap a personal project of mine, and this definitely fills a gap in the space of getting started with Django + Intertia. I'm a Developer with several years of experience in various languages and frameworks, I use Django mostly for personal projects. I am interested in improving developer experience. A couple of ideas that I wanted to open up a discussion on, thoughts very much welcome

1. Provide users an option to disable the `welcome_page` view from their application. An alternative would be to create a `breeze` app in the users project, which they can then use, remove, modify as they please. Its great to have this straight out the box, but users may have their own preferences on how they want to structure their projects
2. Great an option to output the settings that are created at runtime into a settings file, this would be super userful if folks wanted to scaffold a project using `django-breeze` but don't want the dependency to remain going forward.

**Reply by Louxsdon** [reply]
Thanks @mrfolksy for coming up with these sugguestions.

Contributions are very much welcome to make this package great and to improve the development process for developers so feel free to open PRs on whatever you think can make this package work.
